app/jobs/bulk_action_jobs.rb

```diff
@@ -134,6 +134,9 @@ def perform(documents, _params)
       successes = []
       documents.each do |id|
         media_object = MediaObject.find(id)
+        supplemental_files = media_object.supplemental_files
+        DeleteChildFiles.perform_later(supplemental_files, nil)
```

Comment:
I think we can skip the overhead of enqueuing new jobs here and run this inline since we're already inside a background job.
```suggestion
        DeleteChildFiles.perform_now(supplemental_files, nil)
```